peekingduck/pipeline/nodes/output/screen.py

```diff
@@ -49,15 +51,33 @@ def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
         cv2.namedWindow(self.window_name, cv2.WINDOW_NORMAL)
         cv2.moveWindow(self.window_name, self.window_loc["x"], self.window_loc["y"])
+        self.aspect_ratio_thres = 5e-3
```

Comment:
I don't think this is ever modified, consider making it a constant instead.

Reply:
Amended

Reply:
You can make these a module level constant similar to https://github.com/aimakerspace/PeekingDuck/blob/main/peekingduck/pipeline/nodes/dabble/keypoints_to_3d_loc.py

Reply:
amended